### ISSUE TITLE
Journalism: LevelTime

### ISSUE THREAD

**Issue by LswaN58** [opengamedata/opengamedata-core#109]
There is an existing feature named `TotalLevelTime`. We should rename this to `LevelTime`, and rewrite this feature based on the below:

This should be a per-level feature with two subfeatures.
The "main" feature is the "total" time spent on the given level, which is the time from first event to last event. This is currently what is being calculated by the exiting `TotalLevelTime` feature.
The subfeatures are the idle time (refer to existing full-session time feature, named `PlayTime`, for how to access the idle time) and "active" time (total time minus the idle time).

In summary:
* **Main feature**: no special name, calculate by last event timestamp - first event timestamp
* **Idle**: name `Idle`, refer to existing feature
* **Active**: name `Active`, calculate by Total - Idle
